### ISSUE TITLE
Debug new (apparently spurious) Cypress accessibility test failures

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#11761]
## Description
We've been encountering [some apparently unrelated but reasonably reliable accessibility test failures](https://dsva.slack.com/archives/C52CL1PKQ/p1669834551532619) across various PRs, so I need to figure out why and fix the issue.

## Acceptance Criteria
- [ ] PRs are no longer failing accessibility tests for things they didn't change.

**Comment by ndouglas** [comment]
CAUGHT ON VIDEO: brief period of low contrast.  Apparently that's sufficient.  I don't know why it seems to be happening on some node forms and not others, or why this has just started.  Digging continues.

![Screen Shot 2022-12-01 at 8 46 41 AM](https://user-images.githubusercontent.com/1318579/205068928-d350dce9-8374-4385-9359-93fe72962b89.png)

**Comment by ndouglas** [comment]
I wonder if [these CSS transitions](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/11729/files#) are to blame.
